packages/muriel-style/layout.scss

```diff
@@ -0,0 +1,32 @@
+$layout-grid-columns-large: 12;
+$layout-grid-columns-medium: 8;
+$layout-grid-columns-small: 4;
+$layout-grid-gutter-large: 24px;
+$layout-grid-gutter-medium: 24px;
+$layout-grid-gutter-small: 16px; 
+$layout-grid-breakpoint-large: 1280px;
+$layout-grid-breakpoint-medium: 600px;
+$layout-grid-breakpoint-small: 0px;
+
+.layout-grid {
+
+	display: grid;
+	margin: 0;
+	grid-gap: $layout-grid-gutter-small;
+	grid-template-columns: repeat( $layout-grid-columns-small, [col-start] minmax( 0, 1fr ) );
+
+	@media ( min-width: $layout-grid-breakpoint-medium ) and ( max-width: $layout-grid-breakpoint-large - 1px ) {
```

Comment:
I think we should reuse the existing breakpoint mixin here: https://github.com/Automattic/wp-calypso/blob/0233ccdc84055a610992d35e409c76b2981be6a3/assets/stylesheets/shared/mixins/_breakpoints.scss